### PR TITLE
Fix for #167 - `set_approval_for_all` fails in Hash mode

### PR DESCRIPTION
**Pull request by davidatwhiletrue**

### CODE REVIEW

File: tests/src/mint.rs [hunk 1]
```diff
@@ -814,7 +814,111 @@ fn should_set_approval_for_all() {
...
+    let actual_operator: Option<Key> = call_entry_point_with_ret(
+        &mut builder,
+        *DEFAULT_ACCOUNT_ADDR,
+        nft_contract_key,
+        runtime_args! {
+            ARG_IS_HASH_IDENTIFIER_MODE => true,
+            ARG_TOKEN_HASH => token_id_hash,
+        },
+        "get_approved_call.wasm",
+        "get_approved",
```
Comment (gRoussac) [inline tests/src/mint.rs]: Hi @davidatwhiletrue, can you please extract those hard coded values in the file into the constants file ? many thanks

**Review by gRoussac — APPROVED** [review]

File: tests/src/mint.rs (hunk 1 above)
Comment (davidatwhiletrue) [reply tests/src/mint.rs]: Done.

Comment (gRoussac) [reply tests/src/mint.rs]: Something is strange in your commit RETURNED_VALUE_STORAGE_KEY is now replacing entry points ?

File: tests/src/mint.rs [hunk 2]
```diff
@@ -178,7 +179,7 @@ fn entry_points_with_ret_should_return_correct_value() {
             ARG_TOKEN_ID => 0u64,
         },
         GET_APPROVED_WASM,
-        "get_approved",
+        RETURNED_VALUE_STORAGE_KEY,
```
Comment (gRoussac) [inline tests/src/mint.rs:182]:
```suggestion
        ENTRY_POINT_GET_APPROVED,
```

**Comment by darthsiroftardis** [comment]
Closing in favor of #164

**Pull request closed without merging**